config/test.exs

```diff
@@ -2,12 +2,18 @@ use Mix.Config
 
 # Configure your database
 config :backend_vagas, BackendVagas.Repo,
-  username: "postgres",
-  password: "postgres",
-  database: "backend_vagas_test",
+  username: System.get_env("POSTGRES_USER"),
+  password: System.get_env("POSTGRES_PSWD"),
+  database: System.get_env("POSTGRES_DATABASE"),
   hostname: "localhost",
   pool: Ecto.Adapters.SQL.Sandbox
 
+if System.get_env("GITHUB_ACTIONS") do
+  config :backend_vagas, BackendVagas.Repo,
+    username: 'postgres',
```

Comment:
usamos apenas aspas duplas no elixir

Reply:
O credo não devia pegar isso? :thinking: 
Vou rever meu plugin aqui do elixir.

Reply:
eu confesso que nem entendi essa condicional, você define acima como variável de ambiente para o ambiente de teste, mas se for no CI, fica hard code, por que as variáveis de ambiente então? vc quer usar env var local?

Reply:
E vou reformular isso também, vou matar esse if e mover essas variavéis pro env do actions.

Reply:
se local é postgres/postgres e no CI também, não vejo necessidade da variável de ambiente, vc vai precisar definir variáveis de ambiente em ambiente local, o que não é uma prática comum no elixir, pelo menos nos projetos que já vi, geralmente no dev/test ficam os valores default hard code e no prod que lê do env var, nunca vi projetos com .env por exemplo, o `get_env` aceita valor default como parâmetro se quiser, ou passo direto mesmo, sem `get_env`

Reply:
Basicamente sim. Pegar usuário e senha do env pra ser dinamico, porém eu não tava achando como por variável de ambiente no actions, aí achei essa forma de colocar hard coded. Entretanto como já descobri como fazer, vou tirar daqui e colocar no yml

Reply:
Local não tá como postgres postgres, eu mudei, mas entendi o ponto, vou mudar a senah do banco local e tirar o env